definitions/tools/hla_consensus.cwl

```diff
@@ -0,0 +1,108 @@
+#! /usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Script to create consensus from optitype and clinical HLA typing"
+requirements:
+    - class: InlineJavascriptRequirement
+    - class: DockerRequirement
+      dockerPull: "python:3.7.4-slim-buster"
+    - class: ResourceRequirement
+      ramMin: 4000
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'hla_consensus.py'
+        entry: |
+                        import sys, os
+    
```

Comment:
I looked through and get it, but could you add a comment here describing how the consensus is being generated, for the next time we have to look back at this code?